installer/pkg/workflow/destroy.go

```diff
@@ -28,6 +48,87 @@ func destroyBootstrapStep(m *metadata) error {
 	return runDestroyStep(m, bootstrapStep)
 }
 
+func destroyWorkersStep(m *metadata) error {
+	kubeconfig := filepath.Join(m.clusterDir, generatedPath, "auth", "kubeconfig")
+
+	// TODO(bison): Should any of these actually be fatal errors?
```

Comment:
Can we remove this note?  Or maybe you're asking for "should the worker-destroy step take a `--force` or `--keep-going` flag?"?

Reply:
Removed.